db/schema.rb

```diff
@@ -45,6 +45,7 @@
     t.string   "opencongress_url"
     t.datetime "created_at",            null: false
     t.datetime "updated_at",            null: false
+    t.string   "official_title"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.